Makefile

```diff
@@ -48,15 +51,27 @@ cli:
 check-rpc:
 	cd core/rpctest && go test
 
-dockerBuildCore:
+python-sdk:
```

Comment:
Would prefer this to be structured more like the existing java & javascript rules.  Include building the python images in the main target as opposed to adding more rules and a dependency

Reply:
So basically to run the python tests I'd have to build all the images for all other examples? That can take a very long time.

Reply:
I would argue that even the Java and JS tests should be performed separately and then run in separate jobs to make the test suite faster.

Reply:
The asymmetry makes it harder to maintain the Makefile because it won't be clear down the road why one sdk is handled differently than the others.

The first build takes an annoyingly long time, but all subsequent builds are fast (docker builds are cached).

It would also be fine to do a finer-grained structure for all three languages if you think it is important to optimize the initial time to build images.  But let's not have one pattern for this sdk and a different pattern for another sdk.

Reply:
for me, a make `dockerBuildCore` against an empty docker registry takes 9:30 (annoyingly long).  But running it a second time takes 7 seconds.  So lousy initial experience, but the steady state is fine.

Reply:
I completely agree on avoiding asymmetry. I can implement the separation for the other SDKs no problem. That will mean that the image building for the 3 SDKs can proceed in parallel.